faf-java-server-app/src/main/java/com/faforever/server/matchmaker/MatchMakerService.java

```diff
@@ -281,13 +282,13 @@ private void notifyPlayers(MatchMakerSearch search) {
       .forEach(match -> clientService.sendMatchmakerNotification(match.poolName, match.rightPlayer));
   }
 
-  private void createMatchInternal(String title, List<MatchParticipant> participants, MapVersion map, String featuredMod,
+  private void createMatchInternal(String title, List<GameParticipant> presetParticipants, MapVersion map, String featuredMod,
                                    @Nullable UUID requestId, @Nullable ConnectionAware requester) {
-    log.debug("Creating match '{}' with '{}' participants on map '{}'", title, participants.size(), map);
+    log.debug("Creating match '{}' with '{}' presetParticipants on map '{}'", title, presetParticipants.size(), map);
 
-    List<Player> players = participants.stream()
-      .map(matchParticipant -> playerService.getOnlinePlayer(matchParticipant.getId())
-        .orElseThrow(() -> new RequestException(requestId, ErrorCode.PLAYER_NOT_AVAILABLE_FOR_MATCHMAKING_OFFLINE, matchParticipant.getId())))
+    List<Player> players = presetParticipants.stream()
+      .map(gameParticipant -> playerService.getOnlinePlayer(gameParticipant.getId())
+        .orElseThrow(() -> new RequestException(requestId, ErrorCode.PLAYER_NOT_AVAILABLE_FOR_MATCHMAKING_OFFLINE, gameParticipant.getId())))
       .peek(player -> Requests.verify(player.getCurrentGame() == null, requestId, ErrorCode.PLAYER_NOT_AVAILABLE_FOR_MATCHMAKING_OFFLINE, player))
```

Comment:
I know that is not part of thechange, but why is it PLAYER_NOT_AVAILABLE_FOR_MATCHMAKING_OFFLINE, it seems to be rather PLAYER_ALREADY in a game right?

Reply:
if `getOnlinePlayer()` doesn't return the player the he's not online. sounds correct?

Reply:
I mean this line `.peek(player -> Requests.verify(player.getCurrentGame() == null, requestId, ErrorCode.PLAYER_NOT_AVAILABLE_FOR_MATCHMAKING_OFFLINE, player))`

Reply:
Ah, missed that one on my phone. You're right. #100